### ISSUE TITLE
Missing "monetaryMass" in blocks received by websockets

### ISSUE THREAD

**Issue by Insoleet** [ucoin-io/ucoin#343]
It causes validation errors in sakia : 

```
DEBUG:node:connect_current_block:'monetaryMass' is a required property

Failed validating 'required' in schema:
    {'properties': {'certifications': {'items': {'type': 'string'},
                                       'type': 'array'},
                    'currency': {'type': 'string'},
                    'dividend': {'type': ['number', 'null']},
                    'excluded': {'items': {'type': 'string'},
                                 'type': 'array'},
                    'hash': {'type': 'string'},
                    'identities': {'items': {'type': 'string'},
                                   'type': 'array'},
                    'issuer': {'type': 'string'},
                    'joiners': {'items': {'type': 'string'},
                                'type': 'array'},
                    'leavers': {'items': {'type': 'string'},
                                'type': 'array'},
                    'medianTime': {'type': 'number'},
                    'membersCount': {'type': 'number'},
                    'monetaryMass': {'type': 'number'},
                    'nonce': {'type': 'number'},
                    'number': {'type': 'number'},
                    'previousHash': {'type': ['string', 'null']},
                    'previousIssuer': {'type': ['string', 'null']},
                    'signature': {'type': 'string'},
                    'time': {'type': 'number'},
                    'transactions': {'items': {'properties': {'currency': {'type': 'string'},
                                                              'inputs': {'items': {'type': 'string'},
                                                                         'type': 'array'},
                                                              'issuers': {'items': {'type': 'string'},
                                                                          'type': 'array'},
                                                              'outputs': {'item': {'type': 'string'},
                                                                          'type': 'array'},
                                                              'signatures': {'type': 'array'},
                                                              'version': {'type': 'number'}},
                                               'required': ['signatures',
                                                            'version',
                                                            'currency',
                                                            'issuers',
                                                            'inputs',
                                                            'outputs'],
                                               'type': 'object'},
                                     'type': 'array'},
                    'version': {'type': 'number'}},
     'required': ['version',
                  'currency',
                  'nonce',
                  'number',
                  'time',
                  'medianTime',
                  'dividend',
                  'monetaryMass',
                  'issuer',
                  'previousHash',
                  'previousIssuer',
                  'membersCount',
                  'hash',
                  'identities',
                  'joiners',
                  'leavers',
                  'excluded',
                  'certifications',
                  'transactions',
                  'signature'],
     'type': 'object'}

On instance:
    {'actives': [],
     'certifications': [],
     'currency': 'meta_brouzouf',
     'dividend': 0,
     'documentType': 'block',
     'excluded': [],
     'fork': True,
     'hash': '0000025A2258BA3EAD63A06A89574AF05EC3206B',
     'identities': [],
     'issuer': 'FT8hfXyqvYQpNY3F8SVy2ZfTZtXv15SqXVTqSWn6Jtyz',
     'joiners': [],
     'leavers': [],
     'medianTime': 1454514847,
     'membersCount': 19,
     'nonce': 107325,
     'number': 49532,
     'parameters': '',
     'powMin': 5,
     'previousHash': '0000061D5F2775861F659372449A3976EF0A08A1',
     'previousIssuer': '5ocqzyDMMWf1V8bsoNhWb1iNwax1e9M7VTUN6navs8of',
     'raw': 'Version: 1\n'
            'Type: Block\n'
            'Currency: meta_brouzouf\n'
            'Nonce: 107325\n'
            'Number: 49532\n'
            'PoWMin: 5\n'
            'Time: 1454529247\n'
            'MedianTime: 1454514847\n'
            'Issuer: FT8hfXyqvYQpNY3F8SVy2ZfTZtXv15SqXVTqSWn6Jtyz\n'
            'PreviousHash: 0000061D5F2775861F659372449A3976EF0A08A1\n'
            'PreviousIssuer: 5ocqzyDMMWf1V8bsoNhWb1iNwax1e9M7VTUN6navs8of\n'
            'MembersCount: 19\n'
            'Identities:\n'
            'Joiners:\n'
            'Actives:\n'
            'Leavers:\n'
            'Excluded:\n'
            'Certifications:\n'
            'Transactions:\n'
            'iehZBgx9PKzqKMLbe/geuiKpL/Lu/J+XMkz1GNKBKXNbNHRDr7Gsalej21SuL3Zq4IMwRYXPD9915ouCRHnPBw==\n',
     'signature': 'iehZBgx9PKzqKMLbe/geuiKpL/Lu/J+XMkz1GNKBKXNbNHRDr7Gsalej21SuL3Zq4IMwRYXPD9915ouCRHnPBw==',
     'time': 1454529247,
     'transactions': [],
     'type': 'Block',
     'version': 1,
     'wrong': False}
```

**Comment by c-geek** [comment]
Already fixed in 0.14.2: try with a node with this version.

For example `metab.ucoin.io/ws/block` does send the monetaryMass field.

**Comment by c-geek** [comment]
I let you close the issue, maybe there is something I missed!
